### ISSUE TITLE
npm install of 4.0.7 failed on linux (debian 9-stretch)

### ISSUE THREAD

**Issue by getify** [mapbox/node-sqlite3#1161]
I am using node@12.2.0 and npm@6.9.0.

I installed the newly released 4.0.7 on my Mac OSX 10.14, and it worked fine. Then I tried to install it on my Debian 9-Stretch server and I got an npm error.

npm reported this:

```cmd
npm ERR! code ERR_INVALID_OPT_VALUE
npm ERR! The value "false" is invalid for option "readableHighWaterMark"
```

Here's the full npm debug log: https://pastebin.com/0BFuJzHb

**Comment by kewde** [comment]
Hi,

I can not replicate this, I'm running Debian 9 as well.

```
user@host:~/projects/mapbox/node-sample$ cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
user@host:~/projects/mapbox/node-sample$ nvm install 12.0.0
Downloading and installing node v12.0.0...
Downloading https://nodejs.org/dist/v12.0.0/node-v12.0.0-linux-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v12.0.0 (npm v6.9.0)
user@host:~/projects/mapbox/node-sample$ nvm use 12.0.0
Now using node v12.0.0 (npm v6.9.0)
user@host:~/projects/mapbox/node-sample$ npm install sqlite3

> sqlite3@4.0.7 install /home/user/projects/mapbox/node-sample/node_modules/sqlite3
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download 
[sqlite3] Success: "/home/user/projects/mapbox/node-sample/node_modules/sqlite3/lib/binding/node-v72-linux-x64/node_sqlite3.node" is installed via remote
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN sample@1.0.0 No description
npm WARN sample@1.0.0 No repository field.

+ sqlite3@4.0.7
added 113 packages from 102 contributors and audited 160 packages in 77.418s
found 0 vulnerabilities

```

**Comment by getify** [comment]
On this same Debian machine, I am able to build from source (master) but not able to install via npm.

**Comment by kewde** [comment]
I suppose you have already tried this but does removing the node_modules and calling npm install help?

**Comment by getify** [comment]
When I delete `node_modules` and do `npm install sqlite3`, I still get a failure to install, but this time it's a different error. I've reported it (against `npm`) here: https://npm.community/t/failure-to-install-a-package-log-says-its-an-npm-error/7375

**Comment by getify** [comment]
FYI: something was fixed in 4.0.8, because it (and 4.0.9) install fine with npm on both my machines. Not sure why 4.0.7 was broken. Shrugs.

**Comment by kewde** [comment]
4.0.8 was a rerelease of 4.0.7 without the .vscode files that got added by accident.
